spec/models/passenger_spec.rb

```diff
@@ -3,125 +3,161 @@
+    it 'sets the status to pending' do
+      passenger.active!
+      expect { call['pending'] }.to change(passenger, :registration_status)
+    end
+
+    it 'sets the status to active' do
+      passenger.pending!
+      expect { call['active'] }.to change(passenger, :registration_status)
+    end
+  end
+
+  describe '#expiration_display' do
+    subject(:call) { passenger.expiration_display }
+
+    it 'returns nil for a permanent' do
+      passenger.update permanent: true
+      expect(call).to be_nil
+    end
+
+    it 'returns the expiration date of the doctors note for temporary passengers' do
+      date = passenger.eligibility_verification.expiration_date
+      expect(call).to eq(date.strftime('%m/%d/%Y'))
+    end
+  end
+
+  describe '#temporary?' do
+    subject(:call) { passenger.temporary? }
+
+    it 'returns true if the passenger is not permanent' do
+      expect(call).to be(true)
+    end
+  end
+
+  describe 'not having a SPIRE' do
+    it 'is not valid' do
+      spireless_passenger = build(:passenger, spire: '')
+      expect(spireless_passenger).not_to be_valid
+    end
+  end
+
+  describe '#rides_expire' do
+    subject(:call) { passenger.rides_expire }
+
+    context 'with a permanent passenger' do
+      it 'returns nil' do
+        passenger.update permanent: true
+        expect(call).to be_nil
       end
     end
 
-    describe 'rides_expire' do
-      context 'permanent passenger' do
-        it 'returns nil' do
-          @passenger.update permanent: true
-          expect(@passenger.rides_expire).to be_nil
-        end
+    context "with a doctor's note that is expired within the grace period" do
+      it 'returns 3 days from the doctors note expiry' do
+        date = 2.days.ago.to_date
+        passenger.eligibility_verification.update(expiration_date: date)
+        expect(call).to eq(3.business_days.since(date))
       end
+    end
 
-      context 'doctors note present' do
-        context 'doctors note is expired within grace period' do
-          it 'returns 3 days from the doctors note expiry' do
-            date = 2.days.ago.to_date
-            @passenger.eligibility_verification.update(expiration_date: date)
-            expect(@passenger.rides_expire).to eq 3.business_days.since(date)
-          end
-        end
-
-        context 'the doctors note is not expired' do
-          it 'returns three business days after the expiration date of the note' do
-            date = 14.days.from_now.to_date
-            @passenger.eligibility_verification.update(expiration_date: date)
-            expect(@passenger.rides_expire).to eq 3.business_days.after(date)
-          end
-        end
+    context "with a non-expired doctor's note" do
+      it 'returns three business days after the expiration date of the note' do
+        date = 14.days.from_now.to_date
+        passenger.eligibility_verification.update(expiration_date: date)
+        expect(call).to eq(3.business_days.after(date))
```

Comment:
apparently these aren't offenses, but they should also probably make use of `before`s? Again, rubocop is appeased so not requested, just pointing it out

Reply:
Yeah, I'm not sure. I sometimes use `before` blocks when both they "make sense" and Rubocop thinks the example is too long, but I don't know if there's a benefit to that indirection in cases when the "before" steps only apply to the one example?